lint.sh

```diff
@@ -23,7 +23,12 @@ if [ "$LINT" == true ]; then
         statsmodels/graphics/tsaplots.py \
         statsmodels/examples/tests/ \
         statsmodels/iolib/smpickle.py \
+        statsmodels/iolib/tests/test_pickle.py \
+        statsmodels/multivariate/pca.py \
+        statsmodels/regression/mixed_linear_model.py \
+        statsmodels/regression/recursive_ls.py \
         statsmodels/regression/tests/test_lme.py \
+        statsmodels/tools/linalg.py \
```

Comment:
A few small nitpicky flake8 things failed after we added all of this. 

I made the fixes in a separate PR that is still WIP-ish. But master build will fail until that's fixed.